apps/osim/tests/test_models.py

```diff
@@ -64,7 +64,7 @@ class CheckDescFactory:
         ("has cwe", "cwe_id", ""),
         ("has unembargo_dt", "unembargo_dt", None),
         ("has source", "source", ""),
-        ("has reported_dt", "reported_dt", None),
+        # ("has reported_dt", "reported_dt", None),
```

Comment:
is there a point to leaving this as a comment instead of just deleting the line?

Reply:
I am actually not sure :smile: I made this test suite more than a year ago and I am already not very well familiar with all the intentions and I just did not want to spend time finding out the reasons why was it added and whether we can remove it entirely. I just made the test working :smile:

Reply:
Maybe the module will be eventually just removed.